selfdrive/car/gm/interface.py

```diff
@@ -86,7 +87,9 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, expe
     else:  # ASCM, OBD-II harness
       ret.openpilotLongitudinalControl = True
       ret.networkLocation = NetworkLocation.gateway
-      ret.radarOffCan = False
+      radar_whitelist = {CAR.ESCALADE_ESV}  # 0x2cb missing from fingerprint even though it has radar
+      radar_missing = 0x2cb not in fingerprint[0] and candidate not in radar_whitelist
+      ret.radarOffCan = radar_missing
```

Comment:
Think it's safe if you remove `radar_missing`, and only set `radarOffCan`. Then below, dashcam any car that's not integrated at the fwdCamera and radarOffCan. Removes the need for a local variable

Reply:
Better to set `radarOffCan = False` in the Escalade sublogic and move dashcam stuff to the bottom, or to leave `radar_whitelist` where it is?